src/components/elements/checkbox/checkbox.njk

```diff
@@ -1,4 +1,21 @@
-<p>
-    <input type="checkbox" id="{{ id if id else name }}" name="{{ name }}"{% if value %} value="{{ value }}"{% endif %} />
+{% if ariaDescribedby === false %}
+    {% set ariaDescribedby = '' %}
+    {% if hint %}
+        {% set ariaDescribedby = name + '-hint' %}
+    {% elif error %}
+        {% set ariaDescribedby = name + '-error' %}
+    {% elif hint and error %}
+        {% set ariaDescribedby = name + '-hint' + " " + name + '-error' %}
+    {% endif %}
+{% endif %}
+
+<div class="field{% if error %} field--error{% endif %}">
+    <input type="checkbox" id="{{ id if id else name }}" name="{{ name }}"{% if value %} value="{{ value }}"{% endif %}{% if ariaDescribedby != '' %} aria-describedby="{{ ariaDescribedby }}"{% endif %} />
```

Comment:
I'm not sure if there is something wrong here, but the example outputs with `ariaDescribedBy=""` even when there is a hint or error.

Reply:
Fixed in 00d8455.